git-radar

```diff
@@ -83,18 +84,30 @@ if [[ -z $@ ]]; then
   echo "  end"
   echo ""
   echo "  Same as the Bash but for fish."
+  echo ""
+  echo "Fetch_t example:"
+  echo "  export PS1=\"\\W\\\$(git-radar --bash --fetch_t 45) \""
   exit
 fi
 while [[ $# > 0 ]];do
   command="$1"
   shift
 
+  # Default fetch of 5 min
   if [[ "$command" == "--fetch" ]]; then
     nohup $dot/fetch.sh >/dev/null 2>&1 &
   fi
+
+  # Custom fetch of whatever time interval the user wants
+  if [[ "$command" == "--fetch_t" ]]; then
+    # Now $1 is the value of fetch_t
+    nohup $dot/fetch.sh $1 >/dev/null 2>&1 &
```

Comment:
If you're going to use `$1` here don't forget to shift it, so that it doesn't come back around next iteration. You might want to check it conforms to a number too, in case someone writes `--fetch_t --foo` and you pass `--foo` to your script.